ops/testing.py

```diff
@@ -190,15 +190,18 @@ def begin_with_initial_hooks(self) -> None:
         """Called when you want the Harness to fire the same hooks that Juju would fire at startup.
 
         This triggers install, relation-created, config-changed, start, and any relation-joined
-        hooks. Based on what relations have been defined before you called begin().  Note that all
-        of these are fired before returning control to the test suite, so if you want to introspect
-        what happens at each step, you need to fire them directly (eg Charm.on.install.emit()).  In
-        your hook callback functions, you should not assume that workload containers are active;
-        guard such code with checks to Container.can_connect().  You are encouraged to test this by
-        setting the global SIMULATE_CAN_CONNECT variable to True.
+        hooks. Note that this does NOT trigger a pebble-ready hook. Based on what relations have
```

Comment:
I think that this new sentence should go after "Based on what relations ..."

Reply:
Good call.  Done.